src/routes/docs/tutorials/apple/step-2/+page.markdoc

```diff
@@ -0,0 +1,33 @@
+---
+layout: tutorial
+title: Create app
+description: Create an Apple SwiftUI app project using Appwrite.
+step: 2
+---
+
+# Create SwiftUI project {% #create-swift-ui %}
+
+Open Xcode and click **Create a new Xcode project**.
+
+Choose your desired project template, for example **iOS App**,  and click **Next**.
+
+Now enter your app **product name** as `Ideas Tracker` and **bundle identifier** as `example.IdeasTracker` and click **Next**. 
+You will need both of these values later when you create your project in the Appwrite console.
+
+Choose a directory for your project in and click **Create** to create your project.
+
+# Add dependencies {% #add-dependencies %}
+
+To add the Appwrite SDK for Apple as a dependency, open the **File** menu and click **Add Packages**.
+
+In the **Package URL** search box, enter https://github.com/appwrite/sdk-for-apple. 
+
+Once the SDK is found, use `5.0.0` as version, select **Up to Next Major Version** as your **Dependency Rule** and click **Add Package**.
```

Comment:
The version is hardcoded?? We need a list of all the places to go to update when we create a new release then. 😬